Tensile/TensileCreateLibrary.py

```diff
@@ -1153,7 +1169,8 @@ def splitExtraParameters(par):
   argParser.add_argument("LogicPath",       help="Path to LibraryLogic.yaml files.")
   argParser.add_argument("OutputPath",      help="Where to write library files?")
   argParser.add_argument("RuntimeLanguage", help="Which runtime language?", choices=["OCL", "HIP", "HSA"])
-  argParser.add_argument("--cxx-compiler",           dest="CxxCompiler",       choices=["hipcc", 'amdclang++'],       action="store", default="amdclang++")
+  argParser.add_argument("--cxx-compiler",           dest="CxxCompiler",       choices=["hipcc", 'amdclang++'], action="store", 
+                         default="amdclang++" if os.name != "nt" else "clang++")
```

Comment:
Out of scope—please remove from this PR.

Reply:
Removed - too many files opened at once...